app/services/profile_fields/add_link_fields.rb

```diff
@@ -3,17 +3,17 @@ class AddLinkFields
     include FieldDefinition
 
     group "Links" do
-      field "Facebook profile URL", :text_field, placeholder: "https://facebook.com/..."
+      field "Facebook URL", :text_field, placeholder: "https://facebook.com/..."
       field "Youtube URL", :text_field, placeholder: "https://www.youtube.com/channel/..."
-      field "StackOverflow profile URL", :text_field, placeholder: "https://stackoverflow.com/users/..."
-      field "LinkedIn profile URL", :text_field, placeholder: "https://www.linkedin.com/in/..."
-      field "Behance profile URL", :text_field, placeholder: "https://..."
-      field "Dribble profile URL", :text_field, placeholder: "https://dribble.com/..."
-      field "Medium profile URL", :text_field, placeholder: "https://..."
-      field "GitLab profile URL", :text_field, placeholder: "https://..."
-      field "Instagram profile URL", :text_field, placeholder: "https://..."
-      field "Mastodon profile URL", :text_field, placeholder: "https://..."
-      field "Twitch profile URL", :text_field, placeholder: "https://..."
+      field "StackOverflow URL", :text_field, placeholder: "https://stackoverflow.com/users/..."
```

Comment:
For consistency's sake I decided to remove the word "profile" from most of these labels. It also makes the profile migration code a bit easier. Apart from that I added some better placeholders.